outages/README.md

```diff
@@ -0,0 +1,55 @@
+## How to manage outages
+
+### Who is responsible for resolving outages and what should be done
+
+- rotating role - new / incorporate to an existing one (Sentry master, community master)
+- identify an outage
+- inform team
+- inform users
+- be responsible for fixing the issue = try to fix it / ask in the team
+- inform when the outage is resolved
```

Comment:
+1, have a way to provide updates to users, example: https://www.githubstatus.com/incidents/fwtx9db6hz2y

in our case, it could easily be just a github issue